.github/workflows/test.yml

```diff
@@ -12,29 +12,35 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+
+env:
+  FORCE_COLOR: '1' # Make tools pretty.
+  TOX_TESTENV_PASSENV: FORCE_COLOR
+  PIP_DISABLE_PIP_VERSION_CHECK: '1'
+  PIP_NO_PYTHON_VERSION_WARNING: '1'
+  PYTHON_LATEST: '3.11'
+
+
 jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v4
+      - uses: actions/setup-python@v4
         with:
-          python-version: '3.8'
-      - uses: pre-commit/action@v2.0.3
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          python-version: ${{ env.PYTHON_LATEST }}
```

Comment:
you may need to run this with '3.11' or add a step that runs `python -Im pip install setuptools`

Reply:
Yeah, I've just pushed a commit to change `PYTHON_LATEST` to 3.11 :see_no_evil: :man_facepalming:

Reply:
GH actions embarrassing me in front of the wizards